BlazarService/src/main/java/com/hubspot/blazar/resources/ModuleBuildResource.java

```diff
@@ -155,6 +168,42 @@ public long getSize() {
     };
   }
 
+  @GET
+  @Path("{id}/log/download")
+  public Response getLog(@PathParam("id") long moduleBuildId) throws URISyntaxException {
```

Comment:
You should capture the exception and throw a WebException with an appropriate message. It will be confusing for a client (e.g. the UI) getting back a UriSyntaxException.